WordPress/src/main/java/org/wordpress/android/ui/posts/PostListCreateMenuFragment.kt

```diff
@@ -0,0 +1,72 @@
+package org.wordpress.android.ui.posts
+
+import android.content.Context
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.FrameLayout
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelProviders
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.bottomsheet.BottomSheetDialog
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import kotlinx.android.synthetic.main.add_content_bottom_sheet.*
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.main.AddContentAdapter
+import org.wordpress.android.viewmodel.posts.PostListCreateMenuViewModel
+import javax.inject.Inject
+
+class PostListCreateMenuFragment : BottomSheetDialogFragment() {
+    @Inject lateinit var viewModelFactory: ViewModelProvider.Factory
+    private lateinit var viewModel: PostListCreateMenuViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        return inflater.inflate(R.layout.add_content_bottom_sheet, container)
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        val recyclerView = view.findViewById<RecyclerView>(R.id.content_recycler_view)
```

Comment:
question, is there any reason why we don't use kotlinx synthetic extension ( `import kotlinx.android.synthetic....` )  and write this more clearly as in `content_recycler_view.layoutManager = ...` and such? let me know 🙏

Reply:
Nope, I do use them. This was an oversight because I got this logic from the other implementation. Will resolve. 🙏

Reply:
Resolved a344477